README.md

```diff
@@ -142,15 +143,11 @@ Use initialized instance of the Chat class and sends a message using that instan
 
 - Send Message
 ```
-self.chatInstance.sendMessage(message: "Your Message Here") { status, error in
-    if let error = error {
-        print("Error occurred: \(error.localizedDescription)")
+self.chatInstance.sendMessage(message: newMessage, completion: {status, error in
+    if status {
+        print("Message Send Successfully", status)
     } else {
-        if status {
-            print("Message sent successfully!")
-        } else {
-            print("Failed to send message.")
-        }
+        print("Error occurred: \(error.localizedDescription)")
```

Comment:
In errors, can we always have proper log output with error?
So this ^^ like
`print("Message Sending Failed: \(error.localizedDescription)")`